client/src/pages/cejst.tsx

```diff
@@ -1,5 +1,8 @@
 import React from 'react';
+import {defineMessages} from 'react-intl';
 import {Link} from 'gatsby-plugin-intl';
+import {useIntl} from 'gatsby-plugin-intl';
+import {Grid} from '@trussworks/react-uswds';
```

Comment:
Just curious - how come these are necessary to import now? What about writing these gherkin tests inspired the code changes here?

Reply:
Great question!

The basic Gherkin test currently tests that the title of each page is set correctly. To add the title require refactoring a bit. 
1. moving away from custom layouts to one that uses the Grid system (trussworks)
2. internationalizing the content as this was something that needs to be done and since I was here touching the code decided to wrap it into this branch